### PR TITLE
Update speakers details

### PR DESCRIPTION
**Pull request by beluamat29**
- Showing speakers talk (only one) in speaker details page
- When clicking the speakers name in agenda talks, you're redirected to speaker details page
- When clicking the speakers name in my calendar talks, you're redirected to speaker details page

### CODE REVIEW

File: app/views/agenda/_talk.html.haml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   .card__agenda-resume
     %h2= link_to talk.title, agenda_talk_path(@current_conference.slug, talk)
     - talk.speakers.each do |speaker|
-      %p= talk_speaker_and_company(speaker)
+      .p= link_to talk_speaker_and_company(speaker), speaker_path(id: speaker.id)
```
Comment (matiasmelendi) [inline app/views/agenda/_talk.html.haml]: acá le podríamos sacar el `.p`, no?

File: app/views/speakers/show.html.haml [hunk 2]
```diff
@@ -21,22 +21,40 @@
...
+                .card__agenda-header--tag-list
+                  .tag-outline= talk.room
+                  .tag-orange{style: track_color_style(talk.track)}
+                    =talk.track
+            .card__agenda-resume
+              %h2= link_to talk.title, agenda_talk_path(@current_conference.slug, talk)
+              - talk.speakers.each do |speaker|
+                %p= talk_speaker_and_company(speaker)
+
+-#- if @speaker.previous_talk_1.present?
```
Comment (matiasmelendi) [inline app/views/speakers/show.html.haml]: :warning: commented code :warning:

File: app/views/user_talk_calendar/index.html.haml [hunk 3]
```diff
@@ -23,7 +23,7 @@
               .card__agenda-resume
                 %h2= link_to talk.title, agenda_talk_path(@current_conference.slug, talk)
                 - talk.speakers.each do |speaker|
-                  %p= talk_speaker_and_company(speaker)
+                  .p= link_to talk_speaker_and_company(speaker), speaker_path(@current_conference, id: speaker.id)
```
Comment (matiasmelendi) [inline app/views/user_talk_calendar/index.html.haml]: should we remove .p?

File: db/salesforce-schema.sql [hunk 4]
```diff
@@ -15,21 +15,7 @@ SET check_function_bodies = false;
...
-DROP TRIGGER hc_talk__c_logtrigger ON salesforce.talk__c;
-DROP TRIGGER hc_opportunity_status_trigger ON salesforce.opportunity;
-DROP TRIGGER hc_opportunity_logtrigger ON salesforce.opportunity;
-DROP TRIGGER hc_event_status_trigger ON salesforce.event;
-DROP TRIGGER hc_event_logtrigger ON salesforce.event;
-DROP TRIGGER hc_campaign_status_trigger ON salesforce.campaign;
-DROP TRIGGER hc_campaign_logtrigger ON salesforce.campaign;
-DROP TRIGGER hc_account_status_trigger ON salesforce.account;
-DROP TRIGGER hc_account_logtrigger ON salesforce.account;
+
```
Comment (matiasmelendi) [inline db/salesforce-schema.sql]: I won't push this changes on this PR

Comment (matiasmelendi) [reply db/salesforce-schema.sql]: I'd prefer to create a separate branch and carefully review the script

File: app/views/agenda/_talk.html.haml (hunk 1 above)
Comment (beluamat29) [reply app/views/agenda/_talk.html.haml]: yes

File: app/views/speakers/show.html.haml (hunk 2 above)
Comment (beluamat29) [reply app/views/speakers/show.html.haml]: corrected

File: app/views/user_talk_calendar/index.html.haml (hunk 3 above)
Comment (beluamat29) [reply app/views/user_talk_calendar/index.html.haml]: :+1:

File: db/salesforce-schema.sql (hunk 4 above)
Comment (beluamat29) [reply db/salesforce-schema.sql]: done !

**Pull request merged**